store/dapps.js

```diff
@@ -1,20 +1,29 @@
 import * as constants from '~/plugins/constants'
 import axios from '~/plugins/axios'
 
+function randomSeed () {
+  return Math.floor(Math.random() * 4294967295) + 1
```

Comment:
Range is `0` and `2**32 - 1`, inclusive. Right now you have a range between `1` and `2**32 - 1`

Instead of `4294967295` as magic value, I'd introduce a constant for `UINT_MAX`:

```
const UINT_MAX = 4294967295`
Math.floor(Math.random() * (UINT_MAX + 1))
```